examples/lock-app/efr32/src/LockManager.cpp

```diff
@@ -67,17 +68,23 @@ CHIP_ERROR LockManager::Init(chip::app::DataModel::Nullable<chip::app::Clusters:
 bool LockManager::ReadConfigValues()
 {
     size_t outLen;
-    EFR32Config::ReadConfigValueBin(EFR32Config::kConfigKey_LockUser, reinterpret_cast<uint8_t *>(&mLockUser),
-                                    sizeof(EmberAfPluginDoorLockUserInfo), outLen);
+    EFR32Config::ReadConfigValueBin(EFR32Config::kConfigKey_LockUser, reinterpret_cast<uint8_t *>(&mLockUsers),
+                                    sizeof(EmberAfPluginDoorLockUserInfo) * DOOR_LOCK_MAX_USERS, outLen);
```

Comment:
Is your question if DOOR_LOCK_MAX_USERS changes? I'm not sure why this would change.

Reply:
Yes, I meant if that changes...  Sorry for the lack of clarity above.

Why it could change... that's up to the device vendor, right?  Maybe they have figured out a way to support more users....  At least a comment about the assumptions this code is making might be useful.

Reply:
There is an attribute for this field actually. I changed the code to read from the attribute instead of using DOOR_LOCK_MAX_USERS

Reply:
I don't think using the attribute really affects the "storage value is not the same size as our buffer" thing we have going on, right?

Reply:
I've added a check to ensure that mMaxUsers is not greater than the buffer size